specification/07-theory_operations.adoc

```diff
@@ -512,29 +512,147 @@ VMM ->> Device: Enable IDE for the selected stream
 VMM ->> RootPort: Enable IDE for the selected stream
 ....
 
-=== Interface Assignment
+=== Interface Binding
 
 Once both the SPDM session and the IDE link are secured and established, the
-host supervisor domain manager may directly assign a TDI to a TVM, through the
+host supervisor domain manager may bind a TDI and a TVM together, through the
 `COVH` interface. This is a four steps process:
 
-1. The host supervisor domain manager initiates the interface assignment flow by
-   having the TSM move the TDI into the TDISP `CONFIG_LOCKED` state.
-2. The TVM verifies and accepts the locked TDI into its TCB.
-3. The TVM asks the TSM to move the TDI to the TDISP `RUN` state.
+1. The host supervisor domain manager initiates the interface binding flow by
+   having the TSM move the TDI into the TDISP `CONFIG_LOCKED` state. This is
+   achieved through the `sbi_covh_bind_interface()` `COVH` ABI.
+2. The TVM xref:_tdi_acceptation[verifies and accepts the locked TDI] into its
+   TCB.
+3. The TVM asks the TSM to move the TDI to the TDISP `RUN` state, by calling the
+   `sbi_covg_run_interface()` `COVG` ABI.
 4. The TVM verifies that the TDI is in the TDISP `RUN` state and starts
-   using it.
+   using it. This verification is provided by the
+   `sbi_covg_get_interface_state()` `COVG` ABI.
+
+The next two sections respectively give a detailed description of the overall
+process for binding an interface to a TVM, and one if its most critical steps:
+the TVM decision of accepting or rejecting the TDI into its TCB.
+
+==== Binding Flow
+
+Binding an interface and a TVM together goes through the following steps:
+
+1. For the binding process to succeed, the host supervisor domain manager must
+   first pass each of the bound TDI MMIO ranges to the TVM. It does so by
+   calling the `sbi_covh_add_tvm_device_region()` `COVH` ABI. The TSM services
```

Comment:
I do not find `sbi_covh_add_tvm_device_region` definition in chap 9.

Do you mean `sbi_covh_connect_device` or `sbi_covh_bind_interface` ?

Reply:
> I do not find `sbi_covh_add_tvm_device_region` definition in chap 9.

Should be fixed now.

> Do you mean `sbi_covh_connect_device` or `sbi_covh_bind_interface` ?

I mean `sbi_covh_bind_interface`. Does that make sense to you?

Reply:
yes